message.go

```diff
@@ -21,19 +21,23 @@ const (
 	STREAM     NotifyContentFormatType = "STREAM"
 )
 
+type requestIDSetter interface {
+	SetRequestID(reqID string)
+}
+
 type MessageResponse struct {
+	RequestID string
 	XMLName   xml.Name `xml:"Message" json:"-"`
 	Code      string   `xml:"Code,omitempty" json:"code,omitempty"`
 	Message   string   `xml:"Message,omitempty" json:"message,omitempty"`
-	RequestId string   `xml:"RequestId,omitempty" json:"request_id,omitempty"`
 	HostId    string   `xml:"HostId,omitempty" json:"host_id,omitempty"`
 }
 
 type ErrorResponse struct {
+	RequestId string   `xml:"RequestId,omitempty" json:"request_id,omitempty"`
```

Comment:
Why does this field have xml/json tags and line 29 doesn't?

Reply:
Did not add RequestId for all other responses since the json/xml is not needed/used. I actually did not change the ErrorResponse, just moved the RequestId position up, thus the confusion, reverted the position change.

![image](https://user-images.githubusercontent.com/28563006/62157683-95e93900-b2c2-11e9-9b6f-724ae7a29528.png)

Reply:
Actually, MNS response has RequestId in case of error but not for the happy case, thus the ErrorResponse has the xml/json tag

```
<Error xmlns="http://mns.aliyuncs.com/doc/v1">
  <Code>InvalidQueueName</Code>
  <Message>The queuename you provided is invalid. QueueName should start with alpha and contain only alpha, digit or -.</Message>
  <RequestId>{rid}</RequestId>
  <HostId>http://{aid}.mns.cn-shanghai.aliyuncs.com</HostId>
</Error>
```